CallAutomation_CallLiveTranscription/CallAutomationLiveTranscription/CallAutomation_LiveTranscription.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.AI.OpenAI" Version="1.0.0-beta.9" />
```

Comment:
Any reason to use beta version for OpenAI ?

Reply:
No but i do see only the beta versions of openAI in the [nuget](https://www.nuget.org/packages/Azure.AI.OpenAI/#versions-body-tab)